src/glossary.js

```diff
@@ -89,21 +93,48 @@ function Glossary(terms, selectors) {
   this.addEventListener(document.body, 'keyup', this.handleKeyup.bind(this));
 }
 
-/** Populate internal list.js list of terms */
 Glossary.prototype.populate = function() {
+  var self = this;
+  var terms = this.terms.map(function(term) {
+    return {
+      'term': term.term,
+      'definition': term.definition,
+      'termClass': self.classes.termClass,
+      'definitionClass': self.classes.definitionClass
+    };
+  });
+
+  var itemTemplate = _.template(
+    '<li>' +
+      '<button class="data-glossary-term {{ termClass }}">{{ term }}' +
+      '</button>' +
+      '<div class="{{ definitionClass }}">{{ definition }}</div>' +
+    '</li>',
+    {interpolate: /\{\{(.+?)\}\}/g}
+  );
+
+  terms.forEach(function(term) {
+    this.list.insertAdjacentHTML('beforeend', itemTemplate(term));
+  }, this);
+};
+
+/** Initialize list.js list of terms */
+Glossary.prototype.initList = function() {
+  var glossaryId = this.selectors.body.slice(1);
```

Comment:
Well, if a variable is required to be a class, it seems most reasonable to call it e.g. `listClass` and not `selectors.list`--it's a class, not an arbitrary selector. Or, if we want to keep the current naming, we could throw an error if the user passes in invalid selector values. Better to crash as soon as possible, and for as clear a reason as possible, given bad inputs.